app/views/layouts/_sidebar.html.erb

```diff
@@ -15,7 +15,7 @@
       </p>
       <ul class="menu-list">
         <% page[:subpaths].each do |item| %>
-          <% if url_for(only_path: true) == item[:path] then %>
```

Comment:
Do we need that back?

Reply:
This will be fixed in the next PR for contracts controller, this can go through